tests/OracleTest.php

```diff
@@ -114,4 +114,12 @@ public function test12OracleLimitSkip()
             $c->dsql()->table('foo')->where('bar', 1)->field('baz')->limit(10, 99)->render()
         );
     }
+
+    public function testOracleExists()
```

Comment:
`OracleTest.php` file/tests will be dropped soon Oracle DB will be fully tested in CI, just FYI, we can merge now...

Reply:
Then in testExists we should check for each connection? It is already complaining that Mssql\Query::exists is not tested...

Reply:
yes (and I expect that it is already tested indirectly in atk4/data)